### PR TITLE
Move lepton-netlist to utils/

### PR DESCRIPTION
**Pull request by graahnul-grom**
It seems that our utils look more or less organized now.
Разгребли. :-)

Affects #732.

### CODE REVIEW

**Comment by graahnul-grom** [comment]
`travis` can't handle long file names??

**Comment by graahnul-grom** [comment]
Without the last commit (renaming a file with
long file name) `travis` fails with this error:

```
make[7]: *** No rule to make target 'drc2/net-not-driven-with-dont-check-if-driven-directive-drc2.out', needed by 'drc2/net-not-driven-with-dont-check-if-driven-directive-drc2.log'.  Stop.
```

File: utils/netlist/tests/Makefile.am [hunk 1]
```diff
@@ -55,7 +55,7 @@ input_files = \
 	drc2/duplicated_slot.sch \
 	drc2/gnd-with-nc-directive.sch \
 	drc2/gnd-without-nc-directive.sch \
-	drc2/net-not-driven-with-dont-check-if-driven-directive.sch \
+	drc2/NNDWDCIDD.sch \
```
Comment (vzh) [inline utils/netlist/tests/Makefile.am:58]: Please use lowercase letters.  Such a name looks so aloud :-)

**Review by vzh — APPROVED** [review]

File: utils/netlist/tests/Makefile.am (hunk 1 above)
Comment (graahnul-grom) [reply utils/netlist/tests/Makefile.am]: Done.

**Pull request merged**